Assets/PC2D/Scripts/PlatformerMotor2D.cs

```diff
@@ -1122,6 +1113,7 @@ private void Start()
         SetSlopeDegreeAllowed();
 
         ladderZone = LadderZone.Bottom;
+        _collisionMask = staticEnvLayerMask | movingPlatformLayerMask;

```

Comment:
If we're going to store this then it needs to be responsive incase the layers change during run time (if some script sets a new staticEnvLayerMask ect...)